### DISCUSSION TITLE
unrealsales.io is shutting down, favorite list sorting needed

### DISCUSSION THREAD

**Discussion by UncleFred** [hugoattal/OrbitalMarket#45] [Ideas]
As you may know unrealsales.io offered the ability to create a watchlist that could be filtered and sorted.

Most importantly, you could build a favourites lists and filter by sales. This avoided having to comb through hundreds of pages of Epic market's sale content, and only display what favourite items were on sale. It was incredibly useful.

With the imminent shutdown of unrealsales, we need this functionality for the orbital market. Ideally, this would include some sort of account system, although I know you mentioned this was outside your bandwidth.

Is this a question of money? How much money would you need to develop a proper lists system?

**Comment by hugoattal** [comment]
You can now use the regular filters with your favlist, so if you want to display only your favorites products on sale, you can click on these two buttons:
![image](https://github.com/hugoattal/OrbitalMarket/assets/4563971/973fd84b-7514-46d5-b029-dc569565e0e0)

About the account system, I admit I don't have the bandwidth right now... That's why I'm trying to avoid it with local storage and import/export systems.

> Is this a question of money? How much money would you need to develop a proper lists system?

If you can, I'm always looking for sponsors to help me pay for the hosting of Orbital Market: https://github.com/sponsors/hugoattal/

If your company wants to sponsor the development of an account system or more complex features, I'd be happy to chat with you 🙂.

Orbital Market is a side project I'm willing to spend a little of my time on, but right now, I can't justify putting several days of dev just for the sake of it.

**Reply by UncleFred** [reply]
Wow! That's a huge upgrade. I'm very happy with this.
![Hq3IewXQrE](https://github.com/hugoattal/OrbitalMarket/assets/5650331/272b098a-bc68-477f-bd36-4dbd5cc2d0a9)
Only thing missing might be a new sorting option "Discount" (Ascending/Descending). This way, you can put the 75% off packages at the head of the list.

Great work Hugo!
